### ISSUE TITLE
Does flutter-tizen CLI support hot-reloaded?

### ISSUE THREAD

**Issue by justwrite99** [flutter-tizen/flutter-tizen#12]
I've built fluttr-tizen helloworld code and succeeded running it on tizen tv.
I'd like to dig more but wanna know hot reloaded is possible.

Thanks.

**Comment by swift-kim** [comment]
Yes, you can use the feature if you launch your app using the `flutter-tizen run` command without the `--release` option. This help message will be prompted after launching an app:

```
Waiting for Tizen SM-R800 to report its views...                    11ms
Syncing files to device Tizen SM-R800...                           516ms

Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h Repeat this help message.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).
An Observatory debugger and profiler on Tizen SM-R800 is available at: http://127.0.0.1:34025/wG7X7TSM38k=/
```

**Comment by justwrite99** [comment]
Sounds great. Thank you for the quick response. 👍
